.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
+venv
+*.coverage
+coverage_reports
+.env
```

Comment:
Please don't add a gitignore file to storey. You can add stuff to the template if you like.